internal/serve/graphql/schema/transaction.graphqls

```diff
@@ -0,0 +1,23 @@
+# GraphQL Transaction type - represents a blockchain transaction
+# gqlgen generates Go structs from this schema definition
+type Transaction{
+  hash:            String!
+  toId:            Int64!
```

Comment:
Still not sure I understand why this field is present. Isn't the transaction hash the primary key on the DB anyway?